.vscode/settings.json

```
@@ -0,0 +1,4 @@
+{
+  "python.formatting.provider": "yapf",
```

```suggestion
  "python.formatting.provider": "black",
```
Use `black` as a formatter instead

---

Yessir!